features/product/viewing_product_in_admin_panel/viewing_product_catalog_price_history_for_a_simple_product.feature

```diff
@@ -0,0 +1,28 @@
+@viewing_products
+Feature: Seeing catalog price history for a simple product
+    In order to be aware of historical simple product prices
+    As an Administrator
+    I want to browse the catalog price history of a simple product
+
+    Background:
+        Given the store operates on a single channel in "United States"
+        And the store has a product "Wyborowa Vodka" priced at "$1000.00" in "United States" channel
+        And this product is originally priced at "$1200.00" in "United States" channel
+        And there is a catalog promotion with "winter_sale" code and "Winter sale" name
+        And the catalog promotion "Winter sale" is available in "United States"
+        And it applies on "Wyborowa Vodka" product
+        And it reduces price by "90%"
+        And it is enabled
+        And the "Wyborowa Vodka" product is now priced at "$1200.00" and originally priced at "$1400.00"
+        And I am logged in as an administrator
+        And I disable "Winter sale" catalog promotion
+
+    @todo
+    Scenario: Seeing the catalog price history of a simple product
+        When I access "Wyborowa Vodka" product
+        And I go to the product price history
+        Then I should see 4 logs entries in the catalog price history for "Wyborowa Vodka" variant
+        And there should be a log entry on the 1st position with the "$1200.00" selling price, "$1400.00" original price and datetime of the price change
```

Comment:
IMO it will be:
```suggestion
        And there should be a log entry on the 1st position with the "$1400.00" selling price, "$1400.00" original price and datetime of the price change
```